.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -42,9 +42,9 @@ body:
       description: "Use: `k0s sysinfo`"
       value: |
         <details><summary>`k0s sysinfo`</summary>
-        ```text
+        <pre>
         ➡️ Please replace this text with the output of `k0s sysinfo`. ⬅️
-        ```
+        </pre>
         </details>
```

Comment:
I think all this html is unnecessary.

![image](https://user-images.githubusercontent.com/224971/223565845-ce59671f-56cd-4e81-8b63-6694e6e6af65.png)

Reply:
> I think all this html is unnecessary.

Do you mean that the details tag should be removed completely? I'd say that it's helpful, since the output of `k0s sysinfo` is rather long, and is only required occasionally. Having it displayed in its full length by default would blow up the bug reports quite significantly.

The culprit with the current template is that there needs to be an additional blank line between lines 44 and 45, then it works with the backticks, too. There doesn't seem to be a way to get the backticks working inside the summary tag, though.

The proposed change renders as follows in my browser:

![k0s-sysinfo-template-rendered](https://user-images.githubusercontent.com/1215104/223657265-3a59eeb2-1355-4e27-946a-1cba0e0f7c0e.png)

Live demo:

<details><summary>`k0s sysinfo`</summary>
<pre>
➡️ Please replace this text with the output of `k0s sysinfo`. ⬅️
</pre>
</details>

Reply:
Ah right, it's for the rendering of the final issue, not for the input box